requirements-dev.txt

```diff
@@ -1,6 +1,8 @@
+coverage==5.5
 exabyte-json-include==2020.10.19
 jsonschema==2.6.0
 python-slugify==2.0.1
 PyYAML==5.3.1
+toml==0.10.2
 Unidecode==1.1.1
--e .
+-e .[test]
```

Comment:
Does this mean we are omitting the `tests` folder?

Reply:
No, this means that we're doing an editable install of the package in the current directory, with the "test" extra dependencies specified in the setup.py file under `extras_require["test"]`

Reply:
OK